src/App.js

```diff
@@ -30,7 +37,37 @@ function App() {
   return (
     <div className="page">
       <div className="content">
-        {shownScreen}
+        <Router>
```

Comment:
We could move the router object to wrap the entire app in index.js

Reply:
Good idea! The router now wraps the App component and App.js just has the switch statements